src/arch/riscv/vm.c

```diff
@@ -32,7 +32,7 @@ void vcpu_arch_init(struct vcpu *vcpu, struct vm *vm) {
 void vcpu_arch_reset(struct vcpu *vcpu, vaddr_t entry)
 {
     memset(&vcpu->regs, 0, sizeof(struct arch_regs));
-    
+
```

Comment:
Lookout for these white space modifications. Shouldn't appear in the commit.

Reply:
Fixed